src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -289,6 +268,10 @@ protected override bool ReleaseHandle()
         private SafeSslHandle() : base(IntPtr.Zero, true)
         {
         }
+
+        internal SafeSslHandle(IntPtr validSslPointer, bool ownsHandle) : base(validSslPointer, ownsHandle)
```

Comment:
The base constructor that takes an IntPtr is for the *invalidHandleValue*.  If the handle is valid, it shouldn't be passed into the base constructor.